### ISSUE TITLE
[Command][Thumbnails] Make batch-size configurable

### ISSUE THREAD

**Issue by fashxp** [pimcore/pimcore#8800]
make batch-size for `pimcore:thumbnails:*` commands configurable. 

Currently, batch size is fixed to 50. Sometimes you want to generate many thumbnails for less than 50 assets ... this results in single-process generation, even there are hundreds of thumbnails generated...

**Comment by brusch** [comment]
Or maybe it would make sense to make the chunks independent of the type ...

**Comment by brusch** [comment]
This is not entirely solved by #8800

The main issue currently is that `\Pimcore\Bundle\CoreBundle\Command\ThumbnailsImageCommand::fetchItems()` is only based on the amount of images and not in combination with thumbnail defintions. 
So the problem is, that e.g. if you have 20 images in your batch but let's say 100 thumbnail definitions, then you'd get just a single process that processes the 100 thumbnails for each of the 20 images (total 2000 thumbnails). However the expected behavior would be that the total amount of expected thumbnails to be generated (2000 in our example) get chunked into batches. So in our example this would be `(100` thumbnail definitions `* 20` images`) / 50 `batch size ` = 40` batches.

So basically I think we just need to create the cartesian product glued with e.g. `~~~` of Asset IDs and available thumbnail definitions in `fetchItems()`.  In `runSingleCommand()` we'd then need to split that product (e.g. `12345~~~exampleCover` again, so that we can process it.
